### ISSUE TITLE
EA: PM-Org Meeting Agenda

### ISSUE THREAD

**Issue by thomasdemoner** [hackforla/expunge-assist#379]
## This issue tracks the running agenda for the PM-Org Meetings

```
# [YEAR-MO-DAY] Agenda and Notes

### Prework
- [ ] Issue for audit Weekly Audit of labels and milestones https://github.com/hackforla/expunge-assist/issues/1543

### Main Items
- [ ] 

### Recurring items: 
 - [ ] Review any issues that are in the new issue approval column on the [project board](https://github.com/orgs/hackforla/projects/77/views/1)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Maria](https://github.com/hackforla/expunge-assist/issues/assigned/mariaweissman) 
       - [ ] [Rachel](https://github.com/hackforla/expunge-assist/issues/assigned/RaeRaeIcky)
       - [ ] [Rick](https://github.com/hackforla/expunge-assist/issues/assigned/rcpeng34)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ]  Make next meeting's agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next meeting's agenda

```

**Comment by thomasdemoner** [comment]
## 18 November 2021 Agenda:
Add your items below!

- [ ] Evaluate [this] agenda issue is formidable.


#### Extra Notes:
- Below?

**Comment by thomasdemoner** [comment]
Adding another comment here to add participants, not sure I should add everyone as assignees: @Andan-Eddy @brandohayes @ExperimentsInHonesty @yumengluo

**Comment by Andan-Eddy** [comment]
Hey @thomasdemoner I am confused by this- did you create an agenda? I don't see it. I am not sure what formidable means

**Comment by thomasdemoner** [comment]
Yeah, this is meant to be the agenda issue. And formidable was definitely the wrong word choice hahaha, I meant something like confirm that this format serves our purposes for the agenda.

**Comment by Andan-Eddy** [comment]
Okay @thomasdemoner I understand- this doesn't quite fit what I meant. Do you have time later today or I can walk you through it at our PM meeting if you will make it

**Comment by ExperimentsInHonesty** [comment]
Agenda for 12/16/21

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Brandon](https://github.com/hackforla/expunge-assist/issues/assigned/brandohayes)
       - [ ] [Andan](https://github.com/hackforla/expunge-assist/issues/assigned/Andan-Eddy)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Yumeng](https://github.com/hackforla/expunge-assist/issues/assigned/yumengluo)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Action items
Brandon and Tom will talk about responsibility for label issue

### Notes:
Brandon met with Durham Brigade and their project "license recovery" indicated they would reach out to the city to ask them if they want the tool we are developing.

**Comment by ExperimentsInHonesty** [comment]
Agenda for 12/16/21

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Brandon](https://github.com/hackforla/expunge-assist/issues/assigned/brandohayes)
       - [ ] [Andan](https://github.com/hackforla/expunge-assist/issues/assigned/Andan-Eddy)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Yumeng](https://github.com/hackforla/expunge-assist/issues/assigned/yumengluo)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### Update (i.e., how did it go?)
- Brandon and Tom will talk about responsibility for label issue #318
- Brandon will make an agenda issue for the development team which has the same type of Accountability and Support check as above.
- Did I get a response? to 
  - Bonnie left notes for Lori and Sam on an issue #366
  - Bonnie asked Yumeng for an update on #325

**Comment by Andan-Eddy** [comment]
- [ ] we should clear "new issue waiting for approval" every week
- [ ] No assignment of issue until out of "new issue waiting for approval"
- [ ] Discussion of Milestones (examples: hackforla website, internship)
        - [ ] roadmap on how to create milestones: (internship) https://github.com/hackforla/internship/issues/54
        - [ ] roadmap on how to create milestones: (community-survey) https://github.com/hackforla/open-community-survey/issues/43
- [ ] Quick audit of label
      - [ ] add issues to project board
- [ ] Final review of team meeting time
- [ ] Discussion of posting for Dev positions- (got 1 15 yr old from India)- still looking for a team lead

**Comment by Andan-Eddy** [comment]
Agenda for 01/06/22

- [ ] introduce new incoming PMs
- [ ] Complete assignments for each PM
- [ ]  Bonnie
 - [ ] Brandon
 - [ ] Andan
 - [ ] Tom
 - [ ] Yumeng

**Comment by thomasdemoner** [comment]
####Agenda for 01/13/22

**Comment by ExperimentsInHonesty** [comment]
# 2022-01-20 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Brandon](https://github.com/hackforla/expunge-assist/issues/assigned/brandohayes)
       - [ ] [Andan](https://github.com/hackforla/expunge-assist/issues/assigned/Andan-Eddy)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Jessica](https://github.com/hackforla/expunge-assist/issues/assigned/jessicaherling)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Bridget](https://github.com/hackforla/expunge-assist/issues/assigned/Bridget14)
- [x] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ]
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting


### Task Items

### Items for next weeks agenda

**Comment by Andan-Eddy** [comment]
Bonnie mentions future products for Expunge Assist can work on.
Put together notes on what NC Brigade does for potential future products

- what a tool that does X be good for you (ie create a national automation product)

**Comment by ValeriyaMetla** [comment]
- Conduct background research about which states require a declaration letter to file an expungement petition

**Comment by ExperimentsInHonesty** [comment]
please add this to the next agenda
- [ ] update this card with the logo and brief description of the project (see [other card](https://github.com/hackforla/UI-UX/projects/2#card-76597765), for how its done)
  - [ ] https://github.com/hackforla/product-management/projects/8#card-75413379

**Comment by thomasdemoner** [comment]
# 2022-01-27 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [x] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [x] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [x] [Brandon](https://github.com/hackforla/expunge-assist/issues/assigned/brandohayes)
       - [x] [Andan](https://github.com/hackforla/expunge-assist/issues/assigned/Andan-Eddy)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Jessica](https://github.com/hackforla/expunge-assist/issues/assigned/jessicaherling)
       - [x] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Bridget](https://github.com/hackforla/expunge-assist/issues/assigned/Bridget14)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] UXR: research plan for usability testing and interviews with stakeholders 
- [ ] UXR: content writing for HfLA discussion 
- [ ] 
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by jessicaherling** [comment]
This issue tracks the running agenda for the PM-Org Weekly Meetings
# [2/3/22] Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Brandon](https://github.com/hackforla/expunge-assist/issues/assigned/brandohayes)
       - [ ] [Andan](https://github.com/hackforla/expunge-assist/issues/assigned/Andan-Eddy)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Jessica](https://github.com/hackforla/expunge-assist/issues/assigned/jessicaherling)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Bridget](https://github.com/hackforla/expunge-assist/issues/assigned/Bridget14)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] UXR: content writing for HfLA discussion
- [ ] 
   - [ ] 

### FYI

- Research team uploaded ad to Github to recruit researchers; Val and Jess have continued to meet with interested Hack for LA members who want to join the team
- UXR: research team working on plan for interviews and usability testing with stakeholders (multi-pronged approach)

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by thomasdemoner** [comment]
This issue tracks the running agenda for the PM-Org Weekly Meetings
# [2/10/22] Agenda and Notes

### Prework

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Brandon](https://github.com/hackforla/expunge-assist/issues/assigned/brandohayes)
       - [x] [Andan](https://github.com/hackforla/expunge-assist/issues/assigned/Andan-Eddy)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Jessica](https://github.com/hackforla/expunge-assist/issues/assigned/jessicaherling)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Bridget](https://github.com/hackforla/expunge-assist/issues/assigned/Bridget14)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] 
   - [ ] 

### FYI

### Notes from Meeting


### Task Items


### Items for next weeks agenda

**Comment by ExperimentsInHonesty** [comment]
# 2022-02-17 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Brandon](https://github.com/hackforla/expunge-assist/issues/assigned/brandohayes)
       - [ ] [Andan](https://github.com/hackforla/expunge-assist/issues/assigned/Andan-Eddy)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Jessica](https://github.com/hackforla/expunge-assist/issues/assigned/jessicaherling)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Bridget](https://github.com/hackforla/expunge-assist/issues/assigned/Bridget14)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ]
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by Andan-Eddy** [comment]
# 2022-03-17 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Brandon](https://github.com/hackforla/expunge-assist/issues/assigned/brandohayes)
       - [ ] [Andan](https://github.com/hackforla/expunge-assist/issues/assigned/Andan-Eddy)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ]Discuss possible future products for design to work on: Andan mentioned gang injunctions
- [ ] Ask Bonnie to give current example of design system- as next assignments for design to create uniformity for project
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by thomasdemoner** [comment]
https://www.hackforla.org/projects/expunge-assist

**Comment by thomasdemoner** [comment]
# 2022-03-17 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Brandon](https://github.com/hackforla/expunge-assist/issues/assigned/brandohayes)
       - [ ] [Andan](https://github.com/hackforla/expunge-assist/issues/assigned/Andan-Eddy)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Thomas:
   - [ ] Discuss possible fits for new PM with Valeryia
   - [ ] Remind myself to check other Teams' Dev wikis in inspiration for our own

- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Thomas didn't get to finish the Miro board transition last week
   - [ ] Bring up possible new tabs for the kanban, for recurring items / agendas
   - [ ] regarding Andans design systems issue about how the product could address different audiences and needs, (HfLA website, Civic Tech index, Design Systems) Bonnie will provide links.
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting
 
### Task Items

### Items for next weeks agenda

**Comment by thomasdemoner** [comment]
# 2022-03-24 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/w0ahth3re)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Communications guide discussion. This doc is an [example](https://docs.google.com/document/d/1VyC97HUVCj7nkV9C5g_6XlO6HKqujLubGWimDKcuSh4/edit#)
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by thomasdemoner** [comment]
https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3

https://www.figma.com/file/EFoSuj1b9G4aZ7bN8OX8tf/CivicTechIndex---Master-Design-File?node-id=1329%3A11944

**Comment by thomasdemoner** [comment]
# 2022-03-31 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/w0ahth3re)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ]
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by ValeriyaMetla** [comment]
# 2022-04-08 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/w0ahth3re)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### Meeting Items
- [ ] Research team is ready to review internal testing research plan & guide 
   - [ ] Please review #507 (document is attached)
   - [ ] The recruitment message is in the comment to this issue (also send Tom on Slack)
   - [ ] Looking forward to the feedback!
- [ ] Discuss how to use / determine next milestones

### FYI

### Notes from Meeting
 
### Task Items

### Items for next week's agenda

**Comment by ExperimentsInHonesty** [comment]
I need to delete this card 
https://github.com/hackforla/UI-UX/projects/3#card-76604465

and replace it with this one 
https://github.com/hackforla/marketing/projects/4#card-80541066

because I put it on the wrong board.  It goes on the Marketing board

**Comment by thomasdemoner** [comment]
# 2022-04-13 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/w0ahth3re)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Bonnie:
   - [ ] I need to delete this card: https://github.com/hackforla/UI-UX/projects/3#card-76604465
   - [ ] and replace it with this one: https://github.com/hackforla/marketing/projects/4#card-80541066
   - [ ] because I put it on the wrong board. It goes on the Marketing board.
- [ ] Research team is ready to review internal testing research plan & guide 
   - [ ] Please review #507 (document is attached)
   - [ ] The recruitment message is in the comment to this issue (also send Tom on Slack)
   - [ ] Looking forward to the feedback!
- [ ] Discuss how to use / determine next milestones
- [ ] Thomas:
    - [ ] Check in on updating the meeting times on the website
    - [ ] The version that's up is ready for internal testing, just need to update with graphic for the landing page.
    - [ ] Need to put up listing for open PM role as well, to help with the content writing team which might be growing to 3 members.
    - [ ] Should we hide the agendas for old meetings?
    - [ ] Any concerns with minors accessing the tool?
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by ExperimentsInHonesty** [comment]
https://github.com/hackforla/UI-UX/projects/7#card-80674431

**Comment by jamiesiu** [comment]
Basketball

**Comment by jamiesiu** [comment]
# 2022-04-28 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/w0ahth3re)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] 
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting
* Internal Usability Testing - 6 sessions
    * Analysis to be done, discussing plans Friday. estimating 2 weeks. Dev asked for feedback to be digestible
    * Val mentioned to their work and possible they may be able to participate in next round of testing
    * 
### Task Items

### Items for next weeks agenda

**Comment by thomasdemoner** [comment]
# 2022-05-05 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/w0ahth3re)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ]
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by thomasdemoner** [comment]
# 2022-05-12 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ]
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by ValeriyaMetla** [comment]
# 2022-05-19 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Thomas:
   - [ ] Review milestones - ps: I added the team agendas to the "Project Management" milestone. Hope that's correct.
   - [ ] Val: Update on Usability Testing 
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by jamiesiu** [comment]
# [Date] Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ]
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting
Discussed #560 milestones organization
* OCS reference example, red colored boxes to still fill out?
* second sheet "Feature Labels Available" - 
* Next step - ensure each issue is assigned milestone and has matching milestone labels (at least one of the respective feature labels)
* How distribute work of milestones assignments? - by column or by team/discipline?
* 
### Task Items

### Items for next weeks agenda

**Comment by jamiesiu** [comment]
# 2022.06.02 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ]
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting
* More PM's joining HFLA, Green Earth OS project will be closing and some of their PM's may join 
* 
### Task Items

### Items for next weeks agenda

**Comment by ExperimentsInHonesty** [comment]
Add to next agenda (at the beginning).  Does your dev team need more than 1 hour for their meetings?  If they don't please let me know, and we will move their meeting to zoom 3 and move food oasis to zoom 4 because food oasis needs 2 hours.

**Comment by ValeriyaMetla** [comment]
# 2022.06.16 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Translate report to a spreadsheet to clarify actions and prioritization of actions before making issues in GH
   - [ ] 
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by BCHEN222** [comment]
-https://github.com/hackforla/google-drive-migration/issues/8

**Comment by ValeriyaMetla** [comment]
# 2022.07.07 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Translate report to a spreadsheet to clarify actions and prioritization of actions before making issues in GH
   - [ ] Update Bonnie on progress #572
   - [ ] suggestions and next steps 
       - [ ] PMs will review Excel again, ALL issues need to be created, ALL feedback needs to be addressed, only then the issue can be closed. 
- [ ] Project Timeline 
   - [ ] #583
      - [ ] what we need to have before being able to put it in front of users for testing? can we call it "soft launch" or more things need to be done for that?
         - [ ] When changes will made, then another round of internal testing before we bring it before users.
         - [ ] MVP - it works, not perfect. Solving problems, as soon as we do - we can launch
         - [ ] Prioritize in relation for user testing (with actually testing)
   - [ ] #581 
- [ ] Approve GitHUb How-To doc for Wiki
   - [ ] #580 
   - [ ] [Document Kanban Board Management](  https://docs.google.com/document/d/11Fe7mNdmPBP5bD_yLJ1C0_I1TmoK47AuHHrdhdDyWCs/edit#heading=h.nl3p4nf4eqb4)
   - [ ] Check what is missing in Kanban doc and have in our doc - only those things that are missing and are specific for the project 
- [ ] Milestones troubleshooting 
   - [ ] #560 
   - [ ] Where do research is going? Why is it called usability testing (narrow)? Can we rename it?
   - [ ] Milestones seem to reflect early stages of product development (some of the w1), the current issues don't quite fit in current categories. Need support revising those based on the timeline and current priorities (hence goes the need for timeline development). 
- [ ] Zoom background, if can provide examples #575 
- [ ] Research Team has an idea of survey legal professionals about terminology.
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by ExperimentsInHonesty** [comment]
Please add to next agenda - reviewing the changes I made to your email account and next steps
- go through the labels and see if there is a way to tune each account that is sending your messages so its only the messages you need, not spaming the email
- What to so with share requests
- etc.

**Comment by ValeriyaMetla** [comment]
# 2022.07.28 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### Other Recurring Meeting Items 
- [ ] Milestones troubleshooting 
   - [ ] #560 
   - [ ] Where do research is going? Why is it called usability testing (narrow)? Can we rename it?
   - [ ] Milestones seem to reflect early stages of product development (some of the w1), the current issues don't quite fit in current categories. Need support revising those based on the timeline and current priorities (hence goes the need for timeline development). 
- [ ] Translate internal testing report to a spreadsheet to clarify actions and prioritization of actions before making issues in GH #572
   - [ ] PMs will review Excel again, ALL issues need to be created, ALL feedback needs to be addressed, only then the issue can be closed. 
- [ ] Project Timeline #583
   - [ ] PMs will work on creating overarching timeline, PMs will review Excel from usability testing to identify what needs to be prioritized. Should we attach it to milestones? 
- [ ] Approve GitHUb How-To doc for Wiki #580 
   - [ ] [Document Kanban Board Management](  https://docs.google.com/document/d/11Fe7mNdmPBP5bD_yLJ1C0_I1TmoK47AuHHrdhdDyWCs/edit#heading=h.nl3p4nf4eqb4)
   - [ ] Val will check what is missing in Kanban doc and have in our doc - only those things that are missing and are specific for the project 

### New Meeting Items
- [ ] Putting content into WIki. Sam created a document that summarizes the work of the content team. #505 
   - [ ] Review and next steps. Clarify where it will go and any additional resources working with Wiki layout in Github 
- [ ] Research Across States, when is the product is useful? #433
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting
- [ ] Drive Audit: Could you just make copies of the files? 
   - [ ] Old folder will be deleted 
   - [ ] We cannot make copies, need to move the files as need original URL need to be linked, people from Guides team need access to those on regular basis
   - [ ] Public repo WIKI - public facing WIKI
       - [ ] http://www.greenearthos.org/
       - [ ] https://www.mkdocs.org/getting-started/
       - [ ] https://brigade.cloud/

### Task Items

### Items for next weeks agenda

**Comment by ValeriyaMetla** [comment]
# 2022.08.11 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [x] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [x] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [x] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [x] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### Other Recurring Meeting Items 
- [ ] Milestones/Product Timeline 
   - [ ] #560 
   - [ ] Where do research is going? Why is it called usability testing (narrow)? 
   - [ ] Milestones seem to reflect early stages of product development (some of the w1), the current issues don't quite fit in current categories. Need support revising those based on the timeline and current priorities (hence goes the need for timeline development). 
   - [ ] #615 
   - [ ] #583  
- [ ] Google Drive Clean-Up 
- [ ] Competetive/Comparative Analysis 

### New Meeting Items
- [ ] Implementing Sprints - Leads & PMs Meeting
- [ ] UXR Team Leads
   - [ ] Discussion of Partnership Outreach and Interviews with Legal Professionals 
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda
   

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by jamiesiu** [comment]
# [2022.08.18] Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Valeriya](https://github.com/hackforla/expunge-assist/issues/assigned/ValeriyaMetla)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ]
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by jamiesiu** [comment]
# [2023.01.12] Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
       - [ ] Sheron
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Request for feedback on supporting documentation/research roadmaps/research plans/wikis re: short survey approvals
- [ ] HFLA Onboarding 1/16 support (Jamie Siu) / Dev Lead recruiting 
- [ ] 
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by ExperimentsInHonesty** [comment]
Please add to your [onboard/offboard issues](https://github.com/hackforla/expunge-assist/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature%3A+onboard+%2F+offboard%22+) a step to open a ticket on the [website team to update your project card](https://github.com/hackforla/website/issues/new?assignees=&labels=P-Feature%3A+Project+Info+and+Page%2C+role%3A+back+end%2FdevOps%2C+role%3A+front+end%2C+Size%3A+Small%2C+size%3A+0.5pt%2C+time+sensitive&template=project-profile-card-review-and-update.yml&title=Update+Project+Profile%3A+%5BProject+Name%5D) to add or remove a person.

**Comment by jamiesiu** [comment]
# [2023.02.16] Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
       - [ ] Sheron
       - [ ] James
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Check-in on progress with https://github.com/hackforla/expunge-assist/issues/714#issuecomment-1380983087
- [ ] 
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by jccchurch** [comment]
Rescheduling this meeting: https://lettucemeet.com/l/Xv9dN

New Day & Time TBD

NOTE: lettucemeet link is just for the times of the week - ignore the day/month. Meeting will recur monthly as previously scheduled.

**Comment by jccchurch** [comment]
# [2023.06.02] Agenda and Notes

### Roll Call
- [x] Bonnie
- [x] Analicia
- [x] Julia
- [x] Lane
- [ ] Thomas
- [x] Sam

### New Meeting Items (PINNED FROM LAST MEETING
- [ ]  Product Demo with user stories (pick a time)
- [ ] Research recruitment - top of agenda next time
- [ ] Privacy document - how are we protecting data?
- [ ]  What are the pitfalls of having a partner? What is the process? Need to have a deep talk about this - time and attention for that partnership
- [ ] EA Product Vision Board
- [ ] EA Brand/Content Guide Book
- [ ] ChatGPT - how can we utilize AI to make our letters more effective? How can we ensure privacy is maintained? Bonnie will send article - project that tried to train chatGPT

### FYI

### Notes from Meeting

### Items for next meeting agenda

**Comment by jccchurch** [comment]
# [2023.07.28] Agenda and Notes

### Roll Call
- [ ] Bonnie
- [ ] Analicia
- [ ] Julia
- [ ] Lane
- [ ] Thomas
- [ ] Sam

### New Meeting Items (PINNED FROM LAST MEETING
- [ ]  Site hosting & .org email for customer support
      - implications of customer support from our team?
- [ ]  Info & language for Civic Tech Structure
      - not a non-profit... how to call?
- [ ]  Marketing/Branding role
- [ ]  Project Manager role


### FYI

### Notes from Meeting

### Items for next meeting agenda

**Comment by ExperimentsInHonesty** [comment]
Add to next agenda 
- Review zoom setup spreadsheet for a calendar discrepancy BW discovered

**Comment by lmitch10** [comment]
# [2023.08.07] Agenda and Notes

### Roll Call
- [ ] Analicia
- [ ] Julia
- [ ] Lane
- [ ] Thomas

Facilitator: Lane 

### New Meeting Items:
- Review zoom setup spreadsheet for a calendar discrepancy BW discovered 
- HackerOne 
- Lane to write a one pager on HackerOne
- Design updates
- Research updates

- Dev updates
New team members for Dev team

-Discussion on Project manager(should she join our weekly PM meetings?)


### FYI

### Notes from Meeting




### Items for next meeting agenda

**Comment by amejiamesinas** [comment]
# [2023.08.07] Agenda and Notes

### Roll Call
- [ ] Analicia
- [ ] Bonnie
- [ ] Julia
- [ ] Lane
- [ ] Thomas


### New Meeting Items:
- Review zoom setup spreadsheet for a calendar discrepancy BW discovered 
- Onboarding new team members 
- Roadmap Updates
- Q & A
- Question about open source images 
- Question about PM bio for book project  
- Anything else? 


### FYI

### Notes from Meeting




### Items for next meeting agenda

**Comment by ExperimentsInHonesty** [comment]
Please add to the next agenda:
BW: There is a meeting on Friday at 10:30 am for 1 hour, monthly in zoom 1 listed on the [Zoom Setup Spreadsheet](https://docs.google.com/spreadsheets/d/15P3MYyNQ7Xn0pqdrTynDDipjrLl5LCN9xHVLhk9SWa8/edit#gid=0).  However it does not appear in Zoom or on the calendar for your team.

**Comment by amejiamesinas** [comment]
# [2023.09.08] Agenda and Notes

### Roll Call
- [ ] Analicia
- [ ] Bonnie
- [ ] Lane


### New Meeting Items:
- [ ] PM & Team Updates
- [ ] GitHub Changes
- [ ] Calendar cleanup
- [ ] White hat project


### FYI

### Notes from Meeting




### Items for next meeting agenda

**Comment by ExperimentsInHonesty** [comment]
Privacy policy template https://docs.google.com/document/d/1QzN3g8nMDQVNqg0bD7QTS6eGzTlbn725lnHShfU7Kbg/template/preview

**Comment by amejiamesinas** [comment]
To be added to next agenda: Working on weekly label check: question about size (point value and description)

**Comment by luca-navarrete** [comment]
Add to the next agenda: 
```
- [ ] Review with Bonnie 
  - [ ] Slack channels (active/dormant/archived)
  - [ ] Slack channel pins and bookmarks
```

**Comment by amejiamesinas** [comment]
# [2023.09.08] Agenda and Notes

### Roll Call
- [ ] Analicia
- [ ] Bonnie
- [ ] Lane
- [ ] Curtis


### New Meeting Items:
- [ ] New PM Introductions
- [ ] Review with Bonnie 
  - [ ] Slack channels (active/dormant/archived)
  - [ ] Slack channel pins and bookmarks
- [ ] Follow up on: https://github.com/hackforla/product-management/issues/369
  - Weekly actions 
  - Question about size




### FYI

### Notes from Meeting




### Items for next meeting agenda

**Comment by amejiamesinas** [comment]
# [2023.10.04] Agenda and Notes

### Roll Call
- [ ] Analicia
- [ ] Bonnie
- [ ] Lane
- [ ] Curtis


### New Meeting Items:
- [ ] New PM Introductions
- [ ] Archive Slack Channels
- [ ] Follow Up on Updating Team Project Profile: https://github.com/hackforla/website/issues/5627
- [ ] Follow up on: https://github.com/hackforla/product-management/issues/369
  - Project Board
  - Icebox
- [ ] Milestone Question- Team Collabs



### FYI

### Notes from Meeting




### Items for next meeting agenda
- Question about maintainer on onboarding list: https://github.com/hackforla/expunge-assist/issues/456

**Comment by amejiamesinas** [comment]
# [2023.10.25] Agenda and Notes

### Roll Call
- [ ] Analicia
- [ ] Bonnie
- [ ] Lane
- [ ] Curtis


### New Meeting Items:
- [ ] VRMS follow up
- https://github.com/hackforla/expunge-assist/issues/456#issuecomment-1765413307
- [ ] Updating team roster wiki, follow up
-  https://github.com/hackforla/internship/wiki/How-to-activate-inactivate-team-members-from-roster-and-wiki
- [ ] Archive Slack Channels
- [ ] Follow Up on Updating Team Project Profile: https://github.com/hackforla/website/issues/5627
- [ ] Follow up on: https://github.com/hackforla/product-management/issues/369
  - Reviewing Ice box and dependency instructions
- [ ] Research Question: Posting Survey for Internal Research
- [ ] Following up on: https://github.com/hackforla/expunge-assist/issues/868



### FYI

### Notes from Meeting




### Items for next meeting agenda

**Comment by amejiamesinas** [comment]
# [2023.10.25] Agenda and Notes

### Roll Call
- [ ] Analicia
- [ ] Bonnie
- [ ] Lane
- [ ] Curtis
- [ ] Mireya


### New Meeting Items:
- [ ] New Design PM
- [ ] Add PM to Zoom Schedule and VRMS
- [ ] Follow Up on Updating Team Project Profile: https://github.com/hackforla/website/issues/5627
- [ ]  Reviewing Ice box and dependency [instructions](https://docs.google.com/document/d/1LlFnc-rXG_gOS5tw9xWDfgSk05Lva7LHNusXheXTYdA/edit)
- [ ] Research Question: Edits made and Surveys ready to post?
- [Survey 1](https://docs.google.com/forms/d/1D-sv63VENV_mutBHWKuxC5P33pGAmDoYZapdZIC9fnw/edit)
- [Survey 2](https://docs.google.com/forms/d/1Trxv6ejnOhTu7y-ddLTNjUzySt_45Rq8agNhY3GQQL0/edit)
- https://github.com/hackforla/expunge-assist/issues/431#issue-1109651434
- [ ] Last PM-Org meeting for the year-- any other things PM should prioritize before December?

### FYI

### Notes from Meeting



### Items for next meeting agenda

**Comment by amejiamesinas** [comment]
# [2023.11.17] Agenda and Notes

### Roll Call
- [ ] Analicia
- [ ] Bonnie
- [ ] Lane
- [ ] Curtis
- [ ] Mireya


### New Meeting Items:
- [ ] New Year Updates
- [ ] Question about contact us
- [ ] Follow up on Hacker One Project

### FYI

### Notes from Meeting



### Items for next meeting agenda

**Comment by ExperimentsInHonesty** [comment]
# 2024-03-22 Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Tom](https://github.com/hackforla/expunge-assist/issues/assigned/thomasdemoner)
       - [ ] [Jamie](https://github.com/hackforla/expunge-assist/issues/assigned/jamiesiu)
       - [ ] Sheron
       - [ ] James
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] BW: review this if team has made progress, otherwise work on this during the meeting #868
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by amejiamesinas** [comment]
# [Date] Agenda and Notes

### Prework
- [ ] issue for audit Weekly Audit of labels and milestones #318

### Roll Call
- [ ] Analicia
- [ ] Vanessa
- [ ] Sylvia

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.


### New Meeting Items
- [ ] Follow up on Privacy policy
- [ ] Discussion of Target User
- [ ] Discussion of Process 
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next weeks agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next weeks agenda

**Comment by amejiamesinas** [comment]
# [Date] Agenda and Notes

### Prework
- [ ] Issue for audit [Weekly Audit of labels and milestones](https://github.com/hackforla/product-management/issues/369)

### Roll Call

### Recurring items: 
 - [ ] Review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Analicia](https://github.com/hackforla/expunge-assist/issues/assigned/amejiamesinas) 
       - [ ] [Emma](https://github.com/hackforla/expunge-assist/issues/assigned/emmathrash) 
       - [ ] [Vanessa](https://github.com/hackforla/expunge-assist/issues/assigned/vanessaavviles)
       - [ ] [Maria](https://github.com/hackforla/expunge-assist/issues/assigned/mariaweissman) 
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Adding PM's to VRMS
- Maria: maria@mariaweissman.com
- Emma: thrash.emmlorimer@gmail.com
- [ ] Question about website team emergent request’ process
- [ ] Follow up on Privacy Policy
- [ ] Follow up on One Page
- [ ] Plan for next meeting: MVP review

### FYI

### Notes from Meeting

### Task Items

### Items for next meeting's agenda

**Comment by amejiamesinas** [comment]
# [2024-06-14] Agenda and Notes

For this meeting, Bonnie will complete a review of our site and letter generator. 
#1442

**Comment by mariaweissman** [comment]
# [2024-08-09] We are postponing the August meeting, and will meet next in September.

**Comment by sylvia-nam** [comment]
# [2024-09-06] Agenda and Notes

### Prework
- [ ] Issue for audit Weekly Audit of labels and milestones #318

### Roll Call

### Recurring items: 
 - [ ] Review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty
       - [ ] [Emma](https://github.com/hackforla/expunge-assist/issues/assigned/emmathrash) 
       - [ ] [Vanessa](https://github.com/hackforla/expunge-assist/issues/assigned/vanessaavviles)
       - [ ] [Maria](https://github.com/hackforla/expunge-assist/issues/assigned/mariaweissman) 
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Make next meeting's agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next meeting's agenda

**Comment by vanessaavviles** [comment]
# [2024-10-03] Agenda and Notes

### Prework
- [ ] Issue for audit Weekly Audit of labels and milestones https://github.com/hackforla/expunge-assist/issues/1543

### Roll Call

### Recurring items: 
 - [ ] Review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Emma](https://github.com/hackforla/expunge-assist/issues/assigned/emmathrash) 
       - [ ] [Vanessa](https://github.com/hackforla/expunge-assist/issues/assigned/vanessaavviles)
       - [ ] [Maria](https://github.com/hackforla/expunge-assist/issues/assigned/mariaweissman) 
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### New Meeting Items
- [ ] Carry over to next week anything we didn't get covered this week
   - [ ] Ask bonnie about adding new pm rachel to vrms.
   - [ ] Sylvia: Improving collab and project management process presentation. 
   - [ ] Review team charter #1404 with bonnie

### FYI

### Notes from Meeting

### Task Items

### Items for next meeting's agenda

**Comment by ExperimentsInHonesty** [comment]
```
Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.
```

**Comment by mariaweissman** [comment]
Notes: 

- Bonnie: End user is the person whose recording is being expunged. Will likely be used by a legal org, that sits end user down in front of this with the app, instead of responding to their questions
- Bonnie and Rachel will build out setting up research ops guide

**Comment by mariaweissman** [comment]
# [2025-01-03] We are postponing the January 3 meeting, and will meet next in February.

**Comment by SamHyler** [comment]
Bonnie requests this issue be discussed at next meeting (Feb 2025) https://github.com/hackforla/expunge-assist/issues/1664

**Comment by raeraeicky** [comment]
# [2025-02-07] Agenda and Notes

### Prework
- [ ] Issue for audit Weekly Audit of labels and milestones https://github.com/hackforla/expunge-assist/issues/1543

### Main Items
- [ ] Stakeholder 2.0 milestone 70% complete (mix of feedback from stakeholder review 6/2024 and high priority from team)
       - [ ] Needed from Org: #1519 and #1321 
       - [ ] Focus in on High and Medium priority feedback from stakeholder
- [ ] Next steps for the product
       - [ ] What is the next step from HfLA? Finding a community partner or something else?
       - [ ] What is preventing us from doing the next step of finding a community partner in parallel to finishing feedback?
       - [ ] Re-assessment of feedback (some issues may be old and outdated)?
       - [ ] We’re concerned about a lack of movement, usage, and user feedback → volunteer attrition
            - [ ] Also concerned with building product without real world input (is this viable?)
- [ ] Proposal: reassess what's needed for an MVP to start outreach to community partners, and work towards that goal
- [ ] #1664 

### Recurring items: 
 - [ ] Review any issues that are in the new issue approval column on the [project board](https://github.com/orgs/hackforla/projects/77/views/1)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Maria](https://github.com/hackforla/expunge-assist/issues/assigned/mariaweissman) 
       - [ ] [Rachel Ichniowski](https://github.com/hackforla/expunge-assist/issues/assigned/RaeRaeIcky)
       - [ ] [Rick Peng](https://github.com/hackforla/expunge-assist/issues/assigned/rcpeng34)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### FYI

### Notes from Meeting

### Task Items

### Items for next meeting's agenda

**Comment by mariaweissman** [comment]
# 2025-03-07 Postponing March Org Meeting as Team continues to work through #1461

**Comment by mariaweissman** [comment]
# 2025-04-04 Agenda and Notes

### Prework
- [ ] Issue for audit Weekly Audit of labels and milestones https://github.com/hackforla/expunge-assist/issues/1543

### Main Items
- [ ] Our Credits Page Proposal [re Bonnie comment](https://github.com/hackforla/expunge-assist/issues/1442#issuecomment-2168844908): Attribute all icons collectively to Material UI/Google, keep individual credits for illustrations.
> - **Problem**: Google updates their Material UI icon libraries regularly, and there are always small changes. So the icons we are using today won't always appear in their "current" library.
> - **Our ask**: We would like to ask Bonnie if we can remove our icons from the Credits page and instead post a general statement acknowledging Google Material Design UI. We will still keep the credits page, and as requested, we will update direct links and appropriate attributions for the illustrations. We are only asking to remove the icons. 

Reasons:
> - **Google Material Design UI icons do not require attribution**. There are no individual artists to acknowledge. We can meet the need for attribution by posting a statement on the Credits page “all site icons are provided by Google Material UI version XXXX” or something like that. It’s easier to maintain and serves the same purpose. 
> - **Hack for LA Credits Page is not a comparable example to the EA site and how we have managed icons**. We are not pulling icons from multiple places, and our icons were not designed by individual artists requiring attribution. Our site is fundamentally different from the HfLA site.



### Recurring items: 
 - [ ] Review any issues that are in the new issue approval column on the [project board](https://github.com/orgs/hackforla/projects/77/views/1)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Maria](https://github.com/hackforla/expunge-assist/issues/assigned/mariaweissman) 
       - [ ] [Rachel](https://github.com/hackforla/expunge-assist/issues/assigned/RaeRaeIcky)
       - [ ] [Rick](https://github.com/hackforla/expunge-assist/issues/assigned/rcpeng34)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)

### FYI

### Notes from Meeting

### Task Items

### Items for next meeting's agenda

**Comment by raeraeicky** [comment]
# [2025-08-15] Agenda and Notes

### Prework
- [ ] Issue for audit Weekly Audit of labels and milestones https://github.com/hackforla/expunge-assist/issues/1543

### Main Items
- [ ] Updates on Stakeholder 2.0
     - [ ] 3 tickets to complete before review; scheduling
     - [ ] For Bonnie's review: https://github.com/hackforla/expunge-assist/issues/1712
- [ ] New team members
   
### Recurring items: 
 - [ ] Review any issues that are in the new issue approval column on the [project board](https://github.com/orgs/hackforla/projects/77/views/1)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/internship/issues/assigned/ExperimentsInHonesty)
       - [ ] [Maria](https://github.com/hackforla/expunge-assist/issues/assigned/mariaweissman) 
       - [ ] [Rachel](https://github.com/hackforla/expunge-assist/issues/assigned/RaeRaeIcky)
       - [ ] [Rick](https://github.com/hackforla/expunge-assist/issues/assigned/rcpeng34)
       - [ ] [Shradda] (https://github.com/hackforla/expunge-assist/issues/assigned/shraddhasaraf)
       - [ ] [Albert] (https://github.com/hackforla/expunge-assist/issues/assigned/defnotbert)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] Make next meeting's agenda

### FYI

### Notes from Meeting

### Task Items

### Items for next meeting's agenda

**Comment by raeraeicky** [comment]
# [2025-10-03] Agenda and Notes

For this meeting, Bonnie will complete a review of our site and letter generator. See Issue #1809 

- https://github.com/hackforla/expunge-assist/issues/1809
